### PR TITLE
[deckhouse] Fix global values OpenAPI specification

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Fix global values OpenAPI specification.

## Changelog entries
```changes
section: deckhouse
type: fix
summary: Fix global values OpenAPI specification
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: global-hooks/openapi/config-values.yaml [hunk 1]
```diff
@@ -187,12 +187,12 @@ properties:
         properties:
           httpProxy:
             type: string
-            pattern: '^(http|https)://[0-9a-zA-Z\.\-:]+$'
+            pattern: '^http://[0-9a-zA-Z\.\-:]+$'
```
Comment (nabokihms) [inline global-hooks/openapi/config-values.yaml:190]: This change is not valid. 

Http proxy - a proxy for HTTP requests. Https proxy - a proxy for HTTPS requests. You can still serve HTTPS requests with an HTTP proxy and vice versa.

**Review by nabokihms — CHANGES_REQUESTED** [review]

File: global-hooks/openapi/config-values.yaml (hunk 1 above)
Comment (z9r5) [reply global-hooks/openapi/config-values.yaml]: Thnks. Fixed.

**Review by nabokihms — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by z9r5** [comment]
Cherry-picked into release-1.32

**Pull request merged**